README.md

```diff
@@ -1,7 +1,6 @@
 # GDS-SSO
 
-This gem provides everything needed to integrate an application with [Signon]
-(https://github.com/alphagov/signonotron2). It's a wrapper around [OmniAuth](https://github.com/intridea/omniauth) that adds a 'strategy' for oAuth2 integration against Signon,
+This gem provides everything needed to integrate an application with [Signon](https://github.com/alphagov/signonotron2). It's a wrapper around [OmniAuth](https://github.com/intridea/omniauth) that adds a 'strategy' for oAuth2 integration against Signon,
```

Comment:
Thanks for spotting that @GabLeRoux The repo has actually moved now to https://github.com/alphagov/signon . Could you update that and i'll get this merged in? Thanks again :+1: